lib/ansible/module_utils/ec2.py

```diff
@@ -94,9 +94,10 @@ def found(response_code):
 def boto3_conn(module, conn_type=None, resource=None, region=None, endpoint=None, **params):
     try:
         return _boto3_conn(conn_type=conn_type, resource=resource, region=region, endpoint=endpoint, **params)
-    except ValueError:
-        module.fail_json(msg='There is an issue in the code of the module. You must specify either both, resource or client to the conn_type '
-                             'parameter in the boto3_conn function call')
+    except AnsibleAWSError, awsError:
```

Comment:
when do we expect awsError?

edit: oops, got confused by old school python exception syntax. Use new exception syntax (`as e`)